tests/artifacts/pipeline/conda_noversion.yaml.tmpl

```diff
@@ -0,0 +1,5 @@
+channels:
```

Comment:
Since we're gonna start using this one, should we remove the existing `conda.yaml.tmp` file (or rename this one) to avoid confusion in the future?

Reply:
yeah I wasnt sure if it is needed somewhere else. ok